src/main/kotlin/no/nav/tiltaksarrangor/unleash/UnleashService.kt

```diff
@@ -14,4 +14,8 @@ class UnleashService(
 	fun skalLagreAdressebeskyttedeDeltakere(): Boolean {
 		return unleash.isEnabled("amt.enable-adressebeskyttede-deltakere")
 	}
+
+	fun erForslagSkruddPa(): Boolean {
+		return unleash.isEnabled("amt.enable-arrangor-melding")
```

Comment:
kanskje vi bare kan bruke togglen for komet-deltakere så vi kan skru av og på ny løsning kun med en toggle? Må kanskje kombineres med tiltakstypen da.

Reply:
Sant, det må jo kobles med tiltakstype på et vis uanssett.